src/99-appendices/10-file-collections.md

```diff
@@ -24,24 +22,39 @@ included in this appendix:
-| MTS     | flip, mt              | Magnetization transfer saturation          | This method is to calculate a semi-quantitative magnetization transfer saturation index map. The MTS method involves three sets of anatomical images that differ in terms of application of a magnetization transfer RF pulse (MTon or MToff) and flip angle ([Helms et al. 2008](https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.21732)).                                                                                                               |
-| MPM     | flip, mt, echo, part  | Multi-parametric mapping                   | The MPM approaches (a.k.a hMRI) involves the acquisition of highly-similar anatomical images that differ in terms of application of a magnetization transfer RF pulse (MTon or MToff), flip angle and (optionally) echo time and magnitue/phase parts ([Weiskopf et al. 2013](https://www.frontiersin.org/articles/10.3389/fnins.2013.00095/full)). See [here](https://owncloud.gwdg.de/index.php/s/iv2TOQwGy4FGDDZ) for suggested MPM acquisition protocols. |
+Template:
+
+```Text
+sub-<label>/[ses-<label>/]
+    anat/
+        sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>][_rec-<label>][_run-<index>][_echo-<index>][_flip-<index>][_inv-<index>][_mt-<label>][_part-<label>]_<suffix>.nii[.gz]
+```
+
+| Suffix  | Linking entities      | Application                                | Description                                                                                                                                                                                                                                                                                                                                                                                                                         |
+|---------|-----------------------|--------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| VFA     | flip                  | Variable flip angle                        | The VFA method involves at least two spoiled gradient echo (SPGR) of steady-state free precession (SSFP) images acquired at different flip angles. Depending on the provided metadata fields and the sequence type, data may be eligible for DESPOT1, DESPOT2 and their variants ([Deoni et al. 2005](https://doi.org/10.1002/mrm.20314)).                                                                                          |
```

Comment:
All of these suffixes should be represented in the non-parametric table in the MRI page, right? Minus the "linking entities" column and information about the processing algorithms.

Reply:
@tsalo I'd prefer keeping these suffixes in the file collections appendix. In the MRI page (under the parametric images table) there is a link to this table:

> Parametric images listed in the table above are typically generated by
processing a [file collection]... 

should be enough for people who are interested to find them. IMO, adding them to the non-parametric table would create confusion. If it is a necessity to add them to the MRI page, then maybe we can consider a separate table?

Reply:
I'm comfortable doing that in this PR, but I'd like to revisit this when we get around to rendering suffix tables in the specification from the schema (i.e., in a followup to #610). It may not end up being a problem, but my first instinct is that it will make more sense to render all associated suffixes in tables in the main specification text once we have code to do so.